Description: Linkage fixes
 Linker for ubuntu fails to resolve some symbols
 .
 myproxy (6.0rc2-1+gt6.trusty) trusty; urgency=low
 .
   * upgrade to 6.0rc2
Author: Globus Toolkit <support@globus.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- myproxy-6.0rc2.orig/Makefile.am
+++ myproxy-6.0rc2/Makefile.am
@@ -135,13 +135,13 @@ myproxy_store_SOURCES = myproxy_store.c
 
 myproxy_store_LDFLAGS = $(GPT_LDFLAGS)
 
-myproxy_store_LDADD = ./libmyproxy$(_GLOBUS_FLAVOR_NAME).la
+myproxy_store_LDADD = ./libmyproxy$(_GLOBUS_FLAVOR_NAME).la $(LDADD)
 
 myproxy_retrieve_SOURCES = myproxy_get_credential.c
 
 myproxy_retrieve_LDFLAGS = $(GPT_LDFLAGS)
 
-myproxy_retrieve_LDADD = ./libmyproxy$(_GLOBUS_FLAVOR_NAME).la
+myproxy_retrieve_LDADD = ./libmyproxy$(_GLOBUS_FLAVOR_NAME).la $(LDADD)
 
 myproxy_destroy_SOURCES = myproxy_destroy.c
 
@@ -152,7 +152,7 @@ myproxy_destroy_LDADD = ./libmyproxy$(_G
 myproxy_server_SOURCES = myproxy_server.c myproxy_usage.c
 myproxy_server_LDFLAGS = $(GPT_LDFLAGS)
 
-myproxy_server_LDADD = ./libmyproxy$(_GLOBUS_FLAVOR_NAME).la
+myproxy_server_LDADD = ./libmyproxy$(_GLOBUS_FLAVOR_NAME).la $(LDADD)
 
 myproxy_get_trustroots_SOURCES = myproxy_get_trustroots.c
 
@@ -164,13 +164,13 @@ myproxy_get_delegation_SOURCES = myproxy
 
 myproxy_get_delegation_LDFLAGS = $(GPT_LDFLAGS)
 
-myproxy_get_delegation_LDADD = ./libmyproxy$(_GLOBUS_FLAVOR_NAME).la
+myproxy_get_delegation_LDADD = ./libmyproxy$(_GLOBUS_FLAVOR_NAME).la $(LDADD)
 
 myproxy_logon_SOURCES = myproxy_get_delegation.c
 
 myproxy_logon_LDFLAGS = $(GPT_LDFLAGS)
 
-myproxy_logon_LDADD = ./libmyproxy$(_GLOBUS_FLAVOR_NAME).la
+myproxy_logon_LDADD = ./libmyproxy$(_GLOBUS_FLAVOR_NAME).la $(LDADD)
 
 myproxy_change_pass_phrase_SOURCES = myproxy_cp.c
 
